src/model/BorderNode.ts

```diff
@@ -350,7 +355,7 @@ class BorderNode extends Node implements IDropTarget {
       this._addChild(dragNode, insertPos);
     }
 
-    if (this.getSelected() !== -1) { // already open
+    if (select || (select !== false && this.isAutoSelectTab())) {
```

Comment:
For backwards compatibility also need the case for:
(select === undefined && this.isAutoSelectTab() == false && this.getSelected() !== -1)

Reply:
But I think it would make sense to have 2 options for the border:

borderAutoSelectTabWhenOpen       default true
borderAutoSelectTabWhenClosed     default false

since the borders have 2 states: open/closed, currently the border is always open when a tab is selected (so open is equivalent to selected !== -1)

Reply:
Great idea! This is a great way to preserve the existing behavior while being fully configurable.

I've implemented this. The fancy part is I maintained a (I think) cleaner interface for `BorderNode.isAutoSelectTab()` which has an optional argument for whether opened, defaulting to current state.  Let me know what you think.

Reply:
looks good, thanks, I'll merge that